datasets/index.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: default
+title: Data Sources
+nav_order: 9
+has_children: true
+published: false
+---
+
+# Data Sources
+
+Data in Data Commons comes a variety of sources. Each source often includes different surveys. Some sources/surveys include a very large number of variables, some of which might not yet have been imported into Data Commons. The sources have been grouped by category and are listed alphabetically within each category.
+
+1. [United States Data](/datasets/us.html)
+2. [International Data](/datasets/international.html)
+3. [Biomedical Data](/datasets/biomedical.html)
+4. [COVID-19 Data](/datasets/covid19.html)
+5. [Upcoming Data Imports](/datasets/upcoming.html)
```

Comment:
maybe separate this out from the list to differentiate it

Reply:
done